zerver/lib/actions.py

```diff
@@ -4365,7 +4365,36 @@ def user_info(um: UserMessage) -> Dict[str, Any]:
             'id': um.user_profile_id,
             'flags': um.flags_list()
         }
-    send_event(user_profile.realm, event, list(map(user_info, ums)))
+
+    def subscriber_info(user_id: int) -> Dict[str, Any]:
+        return {
+            'id': user_id,
+            'flags': ['read']
+        }
+
+    subscribers_to_be_notified_ids = set()  # type: Set[int]
+    if message.is_stream_message():
+        thread_being_edited = Stream.objects.get(id=stream_id)
```

Comment:
call this `stream_being_edited`; `thread` isn't a concept in Zulip, and if it was, it'd be the topic, not the stream.

Reply:
Also, I think there's a refactor in order.  Above we have:
```
    if message.is_stream_message():
        stream_id = message.recipient.type_id
        event['stream_name'] = Stream.objects.get(id=stream_id).name
```

which already does a database query to fetch the Stream object; we should avoid fetching it twice and instead just save the Stream object in the code above.